deployments/sdm-proxy/values.yaml

```diff
@@ -18,10 +19,10 @@ service:
   # loadBalancerIP:
   annotations:
     # Example annotations for AWS. For more info see https://github.com/kubernetes-sigs/aws-load-balancer-controller
-    # service.beta.kubernetes.io/aws-load-balancer-type: external
-    # service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
-    # service.beta.kubernetes.io/aws-load-balancer-target-group-attributes: preserve_client_ip.enabled=true
-    # service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+    service.beta.kubernetes.io/aws-load-balancer-type: external

```

Comment:
Yeah. I figure if you're on AWS, you need these. And if you're not, they'll be ignored, right?

Reply:
yeah i waffled on this a bit for the on-prem chart. i opted to not add extra annotation "bloat" by toggling them per cloud provider (set by [strongdm.cloud.provider](https://github.com/strongdm/control-plane-helm-chart/blob/main/values.yaml#L70)). but the chart does hardcode some annotations to ensure minimal, required LB configs, like target de-registration and health check settings.

BUT. rather than set all that^ up in this PR, i can tackle it when i take on updating this repo next, i think it's all good for now.